src/i18n/Localizable.xcstrings

```diff
@@ -22222,6 +22338,7 @@
       }
     },
     "Slow" : {
+      "comment" : "a bit ambiguous word. example needed :(",
```

Comment:
"Slow" is the same example as "Fast" before => user want the cursor to be slowly

Reply:
changed noun to adverbs